### PR TITLE
Added a template for Beammp

### PR DESCRIPTION
**Pull request by PaulTimmermann**

### CODE REVIEW

File: beammp/beamng.json [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+      "type": "download"
+    },
+    {
+      "commands": [
+        "rm BeamMP-Server-linux",
+        "unzip BeamMP_Server.zip",
+        "chmod +x BeamMP-Server-linux",
+        "rm BeamMP-Server.exe",
+        "apt update",
+        "apt install liblua5.3-0",
```
Comment (nepcore) [inline beammp/beamng.json]: apt commands are both platform specific (debian, ubuntu and derivatives) and only available to root which PufferPanel doesn't run as in just about every setup, instead mention the dependency in the README file, that gets displayed on the panel and users can handle installing it as correct for their platform

File: beammp/beamng.json [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        "rm LICENSE.txt"
```
Comment (nepcore) [inline beammp/beamng.json]: Removing license files is against the licensing in many cases

**Review by nepcore — CHANGES_REQUESTED** [review]

File: beammp/beamng.json (hunk 2 above)
Comment (PaulTimmermann) [reply beammp/beamng.json]: Yes, but it gets recreated if you start the server just has to be removed to make the istall repeatable and update the config file.

File: beammp/beamng.json (hunk 1 above)
Comment (PaulTimmermann) [reply beammp/beamng.json]: Ok thank you! I will add it to the Readme :)

File: beammp/beamng.json [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  "install": [
+    {
+      "files": [
+        "https://beammp.com/server/BeamMP_Server.zip"
+      ],
+      "type": "download"
+    },
+    {
+      "commands": [
+        "rm BeamMP-Server-linux",
```
Comment (nepcore) [inline beammp/beamng.json]: This will fail on first install as the file does not yet exist

File: beammp/beamng.json (hunk 2 above)
Comment (nepcore) [reply beammp/beamng.json]: You can use the `-u` flag on the unzip command to make it overwrite existing files and create not yet existing ones, that way you don't need to remove any files to make install work in repeat cases

Comment (PaulTimmermann) [reply beammp/beamng.json]: oh thank you very much, i will do so.

**Comment by nepcore** [comment]
I also just realized that you named the json file `beamng.json`, but the directory `beammp`, they need to be named the same to actually work on PufferPanel, so either `beamng/beamng.json` or `beammp/beammp.json`

**Review by nepcore — APPROVED** [review]

**Comment by PaulTimmermann** [comment]
Thank you very much I have learned a lot!

**Pull request merged**